wdl2cwl/WdlV1_1ParserVisitor.py

```diff
@@ -314,6 +323,11 @@ def visitImport_doc(self, ctx:WdlV1_1Parser.Import_docContext):
 
     # Visit a parse tree produced by WdlV1_1Parser#struct.
     def visitStruct(self, ctx:WdlV1_1Parser.StructContext):
+        """Build a WDL struct to be used later when converting to CWL input/output records."""
+        struct = Struct(name=str(ctx.Identifier()))
+        for decl in ctx.unbound_decls():
+            struct.fields.append([str(decl.Identifier()), decl.wdl_type().getText()])
+        self.structs.append(struct)
```

Comment:
So the core of this change depends on saving the structs when visiting the WDL abstract tree. An alternative would be to fetch them when/if necessary from the `tree` (not from `ast`), but I opted for this approach. Happy to change if needed.